action_tutorials/action_tutorials_cpp/README.md

```diff
@@ -0,0 +1,59 @@
+# Action Server
+
+In the constructor for `FibonacciActionServer`, an action server is created with callbacks that are called when a goal is received, when the goal is cancelled and when the goal is accepted:
+
+```
+    this->action_server_ = rclcpp_action::create_server<Fibonacci>(
+      ...
+      "fibonacci",
+      std::bind(&FibonacciActionServer::handle_goal, this, _1, _2),
+      std::bind(&FibonacciActionServer::handle_cancel, this, _1),
+      std::bind(&FibonacciActionServer::handle_accepted, this, _1));
```

Comment:
I think it would be good to remove the leading spaces, here and elsewhere. For example:

```suggestion
his->action_server_ = rclcpp_action::create_server<Fibonacci>(
  ...
  "fibonacci",
  std::bind(&FibonacciActionServer::handle_goal, this, _1, _2),
  std::bind(&FibonacciActionServer::handle_cancel, this, _1),
  std::bind(&FibonacciActionServer::handle_accepted, this, _1));
```